apps/backend/src/applications/applications.controller.ts

```diff
@@ -115,4 +138,24 @@ export class ApplicationsController {
 
     return app.toGetApplicationResponseDTO(apps.length, applicationStep);
   }
+
+  @Patch('/:applicantId')
```

Comment:
I think users should be able to update their own application as well
can leave a TODO for that

Reply:
also the actual events updating should occur here since we can check how many events they have attended and if they have already checked in for the current event